thorlcr/graph/thgraph.cpp

```diff
@@ -2299,6 +2299,7 @@ CFileSizeTracker * CGraphBase::queryTempFileSizeTracker()
     return tempFileSizeTracker.query([] { return new CFileSizeTracker; }, tempFileSizeTrackerCrit);
 }
 
+
```

Comment:
spurious newline, shame as only reason this commit has this file in it. Can you remove?